build.cake

```diff
@@ -116,7 +116,7 @@ DotNetBuildSettings CreateDotNetBuildSettings() =>
         Configuration = configuration,
         NoRestore = true,
         Verbosity = DotNetVerbosity.Minimal,
-        MSBuildSettings = new DotNetMSBuildSettings { Version = packageVersion }
+        MSBuildSettings = new DotNetMSBuildSettings { Version = packageVersion, ContinuousIntegrationBuild = true }
```

Comment:
This flag should be set when build on the server, but not locally. So it needs to be dropped in as a parameter, with default false.

Reply:
I changed the flag to be based on [BuildSystem.GitHubActions.IsRunningOnGitHubActions](https://cakebuild.net/api/Cake.Common.Build.GitHubActions/IGitHubActionsProvider/3F54CBE7)